### PR TITLE
Add failed queues endpoint to health

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?
This PR adds new endpoints to list off the amount of failed jobs in a given queue

## Checklist

- [X] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [X] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/api/health.php [hunk 1]
```diff
@@ -687,6 +688,40 @@
         $response->dynamic(new Document($output), Response::MODEL_HEALTH_ANTIVIRUS);
     });
 
+App::get('/v1/health/queue/failed/:queueName')
```
Comment (christyjacob4) [inline app/controllers/api/health.php:691]:
```suggestion
App::get('/v1/health/queue/failed/:name')
```

File: app/controllers/api/health.php [hunk 2]
```diff
@@ -687,6 +688,40 @@
...
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_HEALTH_FAILED_JOBS)
+    ->inject('response')
+    ->inject('queue')
+    ->action(function (string $queueName, Response $response, Connection $queue) {
+        $client = new Client($queueName, $queue);
+        $failed = $client->countFailedJobs();
+
+        $response->dynamic(new Document(['failed' => $failed]), Response::MODEL_HEALTH_FAILED_JOBS);
```
Comment (christyjacob4) [inline app/controllers/api/health.php:722]: You don't need a new model for this. You can continue to use the `HealthQueue.php` model

Comment (christyjacob4) [reply app/controllers/api/health.php:722]: Also lets format this piece of code

File: src/Appwrite/Utopia/Response/Model/healthFailedJobs.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class HealthFailedJobs extends Model
```
Comment (christyjacob4) [inline src/Appwrite/Utopia/Response/Model/healthFailedJobs.php:8]: we can reuse the existing health response model... so we can remove this

File: app/controllers/api/health.php [hunk 4]
```diff
@@ -687,6 +688,40 @@
...
+        Event::FUNCTIONS_QUEUE_NAME,
+        Event::USAGE_QUEUE_NAME,
+        Event::WEBHOOK_CLASS_NAME,
+        Event::CERTIFICATES_QUEUE_NAME,
+        Event::BUILDS_QUEUE_NAME,
+        Event::MESSAGING_QUEUE_NAME,
+        Event::MIGRATIONS_QUEUE_NAME,
+        Event::HAMSTER_CLASS_NAME
+    ]), 'The name of the queue')
+    ->label('sdk.description', '/docs/references/health/get-failed-queue-jobs.md')
```
Comment (christyjacob4) [inline app/controllers/api/health.php:712]: lets make sure we create this document

File: app/controllers/api/health.php [hunk 5]
```diff
@@ -687,6 +688,40 @@
...
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_HEALTH_FAILED_JOBS)
```
Comment (christyjacob4) [inline app/controllers/api/health.php:715]: we need to update the model here also

File: app/controllers/api/health.php (hunk 2 above)
Comment (PineappleIOnic) [reply app/controllers/api/health.php]: Not sure what you mean by formatting the code exactly, I added some padding to the object array

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**